frontend/src/components/SingleStore.js

```diff
@@ -44,14 +45,18 @@ export default function SingleStore() {
       <h2>Milk</h2>
 
       <StyledProductBox>
```

Comment:
```suggestion
      <ProductBox>
```
For sure it's going to be styled ;)